### DISCUSSION TITLE
Date format

### DISCUSSION THREAD

**Discussion by homerr** [kimai/kimai#4810] [General] [answered]
Morning, hoping this will be simple enough to answer - I've configured all of my timezones etc to where I am (United Kingdom, Europe/London) etc etc but the invoice format on the dates doesn't match what I would expect (DD/MM/YYYY)

What am I doing wrong?

<img width="367" alt="Screenshot 2024-04-22 at 06 38 43" src="https://github.com/kimai/kimai/assets/19776252/5388f027-93b1-409a-bda7-23bd489fa997">
<img width="164" alt="Screenshot 2024-04-22 at 06 39 18" src="https://github.com/kimai/kimai/assets/19776252/fb456ba1-f6cc-4a7a-8daf-bffbddb4fcf3">

Thanks in advance

**Comment by kevinpapst** [comment, marked as answer]
Each invoice template has its own "Language" setting.

**Reply by homerr** [reply]
Thanks @kevinpapst  - I see where I was going wrong now, it was set to "English" but I hadn't bothered to look for "English (United Kingdom)"

Appreciate the guidance, sorry for being a doofus
